src/main/java/tornadofx/App.kt

```diff
@@ -45,30 +49,36 @@ open class App(open val primaryView: KClass<out UIComponent> = NoPrimaryViewSpec
      */
     override val configPath: Path get() = configBasePath.resolve("app.properties")
 
+    val resources: ResourceLookup by lazy { ResourceLookup(this) }
+
     private val trayIcons = ArrayList<TrayIcon>()
-    val resources: ResourceLookup by lazy {
-        ResourceLookup(this)
-    }
 
-    fun <T : FXEvent> fire(event: T) {
-        FX.eventbus.fire(event)
+    init {
+        Stylesheet.importServiceLoadedStylesheets()
+        stylesheet.forEach { importStylesheet(it) }
     }
 
-    constructor(primaryView: KClass<out UIComponent> = NoPrimaryViewSpecified::class, stylesheet: KClass<out Stylesheet>, scope: Scope = FX.defaultScope) : this(primaryView, *arrayOf(stylesheet)) {
+    constructor() : this(NoPrimaryViewSpecified::class)
```

Comment:
Then it could be simplified to just 2 constructors (1 with and 1 without the icon) instead of 4:
```kotlin
open class App(
    open val primaryView: KClass<out UIComponent> = NoPrimaryViewSpecified::class,
    vararg stylesheet: KClass<out Stylesheet>,
    open var scope: Scope = FX.defaultScope
) : Application(), Configurable {

    constructor(
        icon: Image,
        primaryView: KClass<out UIComponent> = NoPrimaryViewSpecified::class,
        vararg stylesheet: KClass<out Stylesheet>,
        scope: Scope = FX.defaultScope
    ) : this(primaryView, *stylesheet, scope = scope) {
        addStageIcon(icon, scope)
    }

    // [...]
}
```